src/PowerShellEditorServices.Hosting/Configuration/EditorServicesConfig.cs

```diff
@@ -17,7 +17,7 @@ public enum ConsoleReplKind
         /// <summary>Use a REPL with the legacy readline implementation. This is generally used when PSReadLine is unavailable.</summary>
         LegacyReadLine = 1,
         /// <summary>Use a REPL with the PSReadLine module for console interaction.</summary>
-        PSReadLine = 2,
+        PSReadLine = 2
```

Comment:
```suggestion
        PSReadLine = 2,
```

Trailing comma is on purpose here